Patch by Sergei Golovan removes unnecessary call to ssl:start() because
ssl module isn't used in ejabberd anymore.

--- ejabberd-2.0.0.orig/src/ejabberd_app.erl
+++ ejabberd-2.0.0/src/ejabberd_app.erl
@@ -39,7 +39,7 @@
     randoms:start(),
     db_init(),
     sha:start(),
-    catch ssl:start(),
+%%  catch ssl:start(),
     stringprep_sup:start_link(),
     translate:start(),
     acl:start(),
